manage-participants/completion_callback.yml

```diff
@@ -48,6 +51,6 @@
             {%- endif -%}
         headers:
           Authorization: Bearer {{ agnosticd_callback_token }}
-        validate_certs: "{{ validate_tower_certs | default(true) }}"
+        validate_certs: "{{ validate_tower_certs | default('yes') }}"
```

Comment:
The quotes should be left off this one since `yes` in this case represents a boolean. Note that Ansible accepts multiple values for booleans to mean the same, e.g.: true, yes, TRUE are all the same, but the string "yes" is generally not (even though it may test to `true` in some cases).